.gitlab-ci.yml

```diff
@@ -1,180 +1,240 @@
 # The testing matrix considers ruby/puppet versions supported by SIMP and PE:
 #
-# https://puppet.com/docs/pe/2017.3/overview/component_versions_in_recent_pe_releases.html
+# https://puppet.com/docs/pe/2018.1/component_versions_in_recent_pe_releases.html
 # https://puppet.com/misc/puppet-enterprise-lifecycle
-# https://puppet.com/docs/pe/2017.3/overview/getting_support_for_pe.html#standard-releases-and-long-term-support-releases
+# https://puppet.com/docs/pe/2018.1/overview/getting_support_for_pe.html
 # ------------------------------------------------------------------------------
-#  release    pup   ruby      eol
-# PE 2017.2   4.10  2.1.9  2018-02-21
-# PE 2017.3   5.3   2.4.1  2018-07
-# PE 2018.1   ???   ?????  ????-??  (LTS)
+# Release       Puppet   Ruby   EOL
+# SIMP 6.1      4.10.6   2.1.9  TBD
+# SIMP 6.2      4.10.12  2.1.9  TBD
+# PE 2016.4.15  4.10.12  2.1.9  2018-12 (LTS)
+# PE 2017.3.10  5.3.8    2.4.4  2018-12 (STS)
+# SIMP 6.3      5.5.7    2.4.4  TBD***
+# PE 2018.1     5.5.6    2.4.4  2020-05 (LTS)***
+# PE 2019.0     6.0      2.5.1  2019-08-31^^^
+#
+# *** = Modules created for SIMP 6.3+ are not required to support Puppet < 5.5
+# ^^^ = SIMP doesn't support 6 yet; tests are info-only and allowed to fail
 ---
-.cache_bundler: &cache_bundler
+stages:
+  - 'sanity'
+  - 'validation'
+  - 'acceptance'
+  - 'compliance'
+  - 'deployment'
+
+image: 'ruby:2.4'
```

Comment:
should this not be 2.4.4?

Reply:
Is there a 2.4.4 image?

Reply:
I honestly had no idea that they provided images down to the Z level. Yes, it probably should.